plugins/module_utils/aci.py

```diff
@@ -1337,41 +1216,15 @@ def get_existing(self):
         and existing configuration will be added to the self.result dictionary.
         """
         uri = self.url + self.filter_string
+        path = self.path + self.filter_string
```

Comment:
Why not handle the addition of `self.filter_string` in you self.api_call() method?

Reply:
self.filter_string is only added for "GET" operations
We would need a separate condition in the api_call() if we want to take care of it there